src/editing/editFeatureComponent.js

```diff
@@ -1282,7 +1282,11 @@ Controller.prototype.openFeatureMenu_ = function (coordinate, features) {
   /** @type {import('ngeo/Menu').MenuActionOptions[]} */
   const actions = [];
   features.forEach((feature) => {
-    const label = feature.get('name') ? feature.get('name') : feature.getId();
+    let label = feature.get('name') ? feature.get('name') : feature.getId();
+    const selectionName = this.editableTreeCtrl.node.metadata['editingSelectionAttribute'];
+    if (selectionName) {
+      label = feature.get(selectionName);
+    }
```

Comment:
A little optimization:
```suggestion
    let label = null;
    const selectionName = this.editableTreeCtrl.node.metadata['editingSelectionAttribute'];
    if (selectionName) {
      label = feature.get(selectionName);
    } else {
      label = feature.get('name') ? feature.get('name') : feature.getId();
    }
```